.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,11 @@
+name: "CodeQL"
+
+on:
+  push:
+  schedule:
+    - cron: '25 13 * * 0'
+
+jobs:
+  codeql:
+    name: CodeQL Analysis
+    uses: canonical/sdcore-github-workflows/.github/workflows/codeql-analysis.yml@main
```

Comment:
Let's use the released version of sdcore-github-workflows (i.e. `v0.0.1`) instead of pointing to `main`. This applies at a couple of places

Reply:
Done